docs/source/setup.rst

```diff
@@ -1,7 +0,0 @@
-setup module
```

Comment:
These are removed as we only want docs generated for code within the `qcp/` directory.